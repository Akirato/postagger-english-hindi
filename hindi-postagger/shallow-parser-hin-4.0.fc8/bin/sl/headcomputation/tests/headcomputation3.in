<Sentence id="1">
1	((	NP	
1.1	bil	NN	<fs af='bil,n,,sg,,d,0,0'>
	))		
2	((	NP	
2.1	awanini	PRP	<fs af='awanu,pn,m,sg,3,,nu,nu'>
2.2	<1+2>/5>	SYM	<fs af='&frasl;,punc,,,,,,'>
	))		
3	((	NP	
3.1	wanani	PRP	<fs af='wAnu,pn,,sg,,,ni,nu'>
	))		
4	((	VGF	
4.1	xveRiswAdu	VM	<fs af='xveRiMcu,v,m,sg,3,,wA,wA'>
4.2	<1+2>/5/	SYM	<fs af='&frasl;,punc,,,,,,'>
4.3	.	SYM	<fs af='&dot;,punc,,,,,,'>
	))		
</Sentence>
<Sentence id="1">
1	((	NP	
1.1	bil	NN	<fs af='bil,n,,sg,,d,0,0'>
	))		
2	((	NP	
2.1	awanini	PRP	<fs af='awanu,pn,m,sg,3,,nu,nu'>
2.2	/	SYM	<fs af='&frasl;,punc,,,,,,'>
	))		
3	((	NP	
3.1	wanani	PRP	<fs af='wAnu,pn,,sg,,,ni,nu'>
	))		
4	((	VGF	
4.1	xveRiswAdu	VM	<fs af='xveRiMcu,v,m,sg,3,,wA,wA'>
4.2	.	SYM	<fs af='&dot;,punc,,,,,,'>
	))		
</Sentence>

