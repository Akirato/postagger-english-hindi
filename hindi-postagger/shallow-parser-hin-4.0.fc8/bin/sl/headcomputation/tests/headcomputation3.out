<Sentence id="1">
1	((	NP	<fs af='bil,n,,sg,,d,0,0' head=bil>
1.1	bil	NN	<fs af='bil,n,,sg,,d,0,0' name=bil>
	))		
2	((	NP	<fs af='awanu,pn,m,sg,3,,nu,nu' head=awanini>
2.1	awanini	PRP	<fs af='awanu,pn,m,sg,3,,nu,nu' name=awanini>
2.2	<1+2>/5>	SYM	<fs af='&frasl;,punc,,,,,,' name=<1+2>/5>>
	))		
3	((	NP	<fs af='wAnu,pn,,sg,,,ni,nu' head=wanani>
3.1	wanani	PRP	<fs af='wAnu,pn,,sg,,,ni,nu' name=wanani>
	))		
4	((	VGF	<fs af='xveRiMcu,v,m,sg,3,,wA,wA' head=xveRiswAdu>
4.1	xveRiswAdu	VM	<fs af='xveRiMcu,v,m,sg,3,,wA,wA' name=xveRiswAdu>
4.2	<1+2>/5/	SYM	<fs af='&frasl;,punc,,,,,,' name=<1+2>/5/>
4.3	.	SYM	<fs af='&dot;,punc,,,,,,' name=.>
	))		
</Sentence>
<Sentence id="1">
1	((	NP	<fs af='bil,n,,sg,,d,0,0' head=bil>
1.1	bil	NN	<fs af='bil,n,,sg,,d,0,0' name=bil>
	))		
2	((	NP	<fs af='awanu,pn,m,sg,3,,nu,nu' head=awanini>
2.1	awanini	PRP	<fs af='awanu,pn,m,sg,3,,nu,nu' name=awanini>
2.2	/	SYM	<fs af='&frasl;,punc,,,,,,' name=/>
	))		
3	((	NP	<fs af='wAnu,pn,,sg,,,ni,nu' head=wanani>
3.1	wanani	PRP	<fs af='wAnu,pn,,sg,,,ni,nu' name=wanani>
	))		
4	((	VGF	<fs af='xveRiMcu,v,m,sg,3,,wA,wA' head=xveRiswAdu>
4.1	xveRiswAdu	VM	<fs af='xveRiMcu,v,m,sg,3,,wA,wA' name=xveRiswAdu>
4.2	.	SYM	<fs af='&dot;,punc,,,,,,' name=.>
	))		
</Sentence>
